AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/RequestFormatter.kt

```diff
@@ -0,0 +1,14 @@
+package com.automattic.android.tracks.crashlogging
+
+import okhttp3.Request
+
+/**
+ * Formats OkHttp requests before using them as a breadcrumb or performance span
+ *
+ * @return formatted url, sent to Sentry
+ */
+interface RequestFormatter {
+    fun formatRequestUrl(request: Request): FormattedUrl
+}
```

Comment:
We want to format the URL to:
- obfuscate it from private data
- make merging the same endpoints easier in Sentry